client/src/app/lounge/pc-info.tsx

```diff
@@ -1,7 +1,39 @@
-export default function PcInfo() {
+import { PC } from "../../interfaces/pc";
+import { checkOutGamer } from "../../services/activity";
+
+interface PCInfoProps {
+  pc: PC;
+  timeRemaining: string;
+  isOccupied: boolean;
+}
+
+const PcInfo: React.FC<PCInfoProps> = ({ pc, timeRemaining, isOccupied }) => {
+  const handleClick = async () => {
+    await checkOutGamer(pc.studentNumber, pc.pcNumber);
+  };
+
   return (
     <div>
       <h1>PC Info</h1>
+      <p>PC Number: {pc.pcNumber}</p>
+      <p>Student Number: {pc.studentNumber}</p>
+      <p>Game: {pc.game}</p>
+      <p>Started At: {pc.startedAt}</p>
+      <p>First Name: {pc.firstName}</p>
+      <p>Last Name: {pc.lastName}</p>
+      <p>Membership Tier: {pc.membershipTier}</p>
+      <p>Notes: {pc.notes}</p>
+      <p>Time Remaining: {timeRemaining}</p>
+      <p>Status: {isOccupied ? "Occupied" : "Available"}</p>
+
+      <button
+        className="rounded bg-blue-500 p-2 text-white"
+        onClick={handleClick}
+      >
+        Free
+      </button>
     </div>
   );
-}
+};
+
+export default PcInfo;
```

Comment:
Lets make `PcInfo` -> `PCInfo`. I think PC has been fully capitalized throughout the codebase, just for consistency 😄